packages/common/src/messages/responses/grpc-response-types.ts

```diff
@@ -57,6 +57,24 @@ export class _ListCachesResponse {
   }
 }
 
+export class _GenerateApiTokenResponse {
+  readonly apiToken: string;
+  readonly refreshToken: string;
+  readonly endpoint: string;
+  readonly validUntil: number;
+  constructor(
+    apiToken?: string,
+    refreshToken?: string,
+    endpoint?: string,
+    validUntil?: number
```

Comment:
what's the motivation for making these nullable?

Reply:
I forgot to update those to be required. Thank you!